### PR TITLE
Change regexp in install script

### PR DESCRIPTION
**Pull request by curquiza**

### CODE REVIEW

File: download-latest.sh [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 # COLORS
-RED="\033[31m"
-GREEN="\033[32m"
-DEFAULT="\033[0m"
+RED='\033[31m'
+GREEN='\033[32m'
+DEFAULT='\033[0m'
 
 # GLOBALS
-REGEXP_SEMVER='v[^0-9]*\([0-9]*\)[.]\([0-9]*\)[.]\([0-9]*\)\([0-9A-Za-z-]*\)'
```
Comment (Kerollmops) [inline download-latest.sh]: What were those `[.]`?

Comment (Kerollmops) [reply download-latest.sh]: Ok I understand, please keep them like that as this is specifying the raw `.`, if you remove the brackets you specify any character. And therefore `v0x6f8` would be accepted.

Comment (curquiza) [reply download-latest.sh]: Brackets are usefull when there are many characters. It means `one of the characters in the brackets`.

Comment (Kerollmops) [reply download-latest.sh]: Yes, but if you remove them therefore you use the `.`, the `.` means `any character you want`. You don't want that, you want the raw `.` character, specifying `[.]` means, `one of the characters in the brackets` and therefore, as there is only one character, `match a point please`.

Comment (curquiza) [reply download-latest.sh]: No, I don't think so:
```bash
$ echo '"v0x6f8.12.1"' | grep '\"v\([0-9]*\).\([0-9]*\).\([0-9]*\)\"'
[nothing]
$ echo '"v0.12.1"' | grep '\"v\([0-9]*\).\([0-9]*\).\([0-9]*\)\"'
"v0.12.1"
```

Comment (curquiza) [reply download-latest.sh]: Maybe I did not understand

Comment (curquiza) [reply download-latest.sh]: (I put back the brackets btw)

Comment (Kerollmops) [reply download-latest.sh]: I think you did not understand, here the example showing what I want to explain:
```bash
echo '"v0x12x1"' | \grep '\"v\([0-9]*\).\([0-9]*\).\([0-9]*\)\"'
"v0x12x1"
```

Comment (Kerollmops) [reply download-latest.sh]: Your previous example showed that `"v0x6f8.12.1"` did not match but it was because you had too many numbers specified.

**Comment by Kerollmops** [comment]
Thank you, great Job! Merging :)

**Pull request merged**